examples/3dbeam.jl

```diff
@@ -0,0 +1,130 @@
+# This file is a part of JuliaFEM.
+# License is MIT: see https://github.com/JuliaFEM/FEMMaterials.jl/blob/master/LICENSE
+
+using JuliaFEM, FEMMaterials, Materials, FEMBase, LinearAlgebra
+import FEMMaterials: Continuum3D, MecaMatSo
+
+# include(joinpath("..","src","mecamatso.jl"))
+# include("idealplastic.jl")
+
+mesh = abaqus_read_mesh(joinpath("plastic_beam.inp"))
+beam_elements = create_elements(mesh, "Body1")
+bc_elements_1 = create_nodal_elements(mesh, "BC1")
+bc_elements_2 = create_nodal_elements(mesh, "BC2")
+trac_elements = create_surface_elements(mesh, "PRESSURE")
+
+update!(beam_elements, "youngs modulus", 200.0e3)
+update!(beam_elements, "poissons ratio", 0.3)
+update!(beam_elements, "yield stress", 100.0)
+update!(beam_elements, "plastic strain", 0.0 => zeros(3,3))
+for j in 1:3
+    update!(bc_elements_1, "displacement $j", 0.0)
+end
+update!(bc_elements_2, "displacement 1", 0.0)
+update!(bc_elements_2, "displacement 2", 0.0)
+update!(trac_elements, "surface pressure", 0.0 => 0.00)
+update!(trac_elements, "surface pressure", 1.0 => 2.70)
+
+# Initialize material model to integration points
+#for element in beam_elements
+#    for ip in get_integration_points(element)
+#        ip.fields["material"] = field(IdealPlastic(element, ip, 0.0))
+#        update!(ip, "stress", 0.0 => zeros(6))
+#         update!(ip, "strain", 0.0 => zeros(6))
+#     end
+# end
+
+beam = Problem(Continuum3D, "plastic beam", 3)
+beam.properties.material_model = :IdealPlastic
+trac = Problem(Continuum3D, "traction", 3)
```

Comment:
@ahojukka5 now the problem is these trac elements, they are also Continuum3D type, but they don't have integration points and material properties etc.